### PR TITLE
utter from website demo

### PR DESCRIPTION
**Pull request by ArjaanBuijk**
- closes https://github.com/RasaHQ/rasa-demo/issues/346
The domain file was updated with utterances from the website-demo.rasa.com instance.

- I tried not to overwrite any changes that were recently made by Ella.
- I trained the model and tested several story flows, using the cmd-line. All seems to behave OK.
- I could not do detailed testing yet, because I don't have a config.py for gdrive & mailchimp

### CODE REVIEW

File: domain.yml [hunk 1]
```diff
@@ -603,39 +724,44 @@ templates:
   utter_search_bar:
   - text: Try using the search bar above for now.
   utter_source_code:
-  - text: You can find my source code [here](https://github.com/RasaHQ/rasa-demo/).
+  - text: "Like the Rasa Stack, my code is completely open source and you can find it [here](https://github.com/RasaHQ/rasa-demo) 👻"
```
Comment (erohmensing) [inline domain.yml]:
```suggestion
  - text: "Like Rasa, my code is completely open source and you can find it [here](https://github.com/RasaHQ/rasa-demo) 👻"
```

File: domain.yml [hunk 2]
```diff
@@ -603,39 +724,44 @@ templates:
...
   utter_spacy:
   - text: 'SpaCy has excellent pre-trained named-entity recognizers for this -- check
       out this [interactive demo](https://demos.explosion.ai/displacy-ent/). I would
-      suggest using the SpacyEntityExtractor component in your pipeline.'
+      suggest using the ner_spacy component in your pipeline.'
```
Comment (erohmensing) [inline domain.yml]:
```suggestion
      suggest using the SpacyEntityExtractor component in your pipeline.'
```

File: domain.yml [hunk 3]
```diff
@@ -234,33 +235,49 @@ templates:
...
-      \ for how I can improve."
+  - text: "I'm sorry I couldn't help you. Please let me know if you have any suggestions for how I can improve."
   utter_ask_for_nlu_specifics:
-  - text: Anything in particular, e.g. intent classification or entity recognition?
+  - text: Anything in particular you'd like to know about, e.g. intent classification 
+      or entity recognition?'
   utter_ask_goal:
-  - text: "Do you want more Rasa in your life? You can subscribe to our newsletter or\
-      \ you can talk to our nice sales team 🧞‍"
+  - text: "If you're new to Rasa, I can help you get started! If you've already started building a bot you can also ask me about the different parts of Rasa 🐦"
```
[the remark this answers is not in the document]
Comment (akelad) [reply domain.yml]: hmm let's for now leave this as is. If we decide to put her on the website, we can have it be a different starting message :)

File: domain.yml [hunk 4]
```diff
@@ -269,88 +286,134 @@ templates:
...
   - text: What kind of bot do you want to build with Rasa?
   utter_ask_weather:
-  - text: "I don't know about where you live, but in my world it's always sunny *"
-  - text: "Honestly it's a bit too hot these days 😅"
-  - text: Blue skies and sunshine!
-  - text: "Let's just say it hasn't rained in weeks *"
+    - text: 'I don''t know about where you live, but in my world it''s always sunny ☀️'
+    - text: It's sunny where I live!
+    - text: 'The flowers are starting to blossom, so it must be spring!'
+    - text: "Spring is here 💐"
```
[the remark this answers is not in the document]
Comment (akelad) [reply domain.yml]: yeah i used to manually update these dependent on season hahah

**Comment by akelad** [comment]
@ArjaanBuijk can you close this and reopen against master? once you've made ellas changes :D since the PR got merged now

File: domain.yml [hunk 5]
```diff
@@ -158,9 +158,9 @@ templates:
   - text: "Looks like {email} is already subscribed - good choice! If you haven't received\
       \ a newsletter in a while, maybe check your spam folder 🗑"
   utter_also_explain_core:
-  - text: Would you also like me to explain Core?
+  - text: Would you also like me to explain Rasa Core?
   utter_also_explain_nlu:
-  - text: Would you also like me to explain NLU?
+  - text: Would you also like me to explain Rasa NLU?
```
[the remark this answers is not in the document]
Comment (ArjaanBuijk) [reply domain.yml]: Ok, that makes sense. I found a couple more places where Rasa NLU & Rasa Core should be replaced with NLU & Core.  I will update these.

File: domain.yml [hunk 6]
```diff
@@ -223,9 +223,10 @@ templates:
   - text: Yes, that is possible! You can connect Rasa to any channel you like. You can
       find more information [here](https://rasa.com/docs/core/connectors/).
   utter_ask_faq_what_is_forum:
-  - text: "Our [forum](https://forum.rasa.com) is the place where all the community\
-      \ buzz happens. Join discussions, get the answers to your technical questions\
-      \ and stay up-to-date with what's happening at Rasa."
+  - text: >-
+      [Rasa Forum](https://forum.rasa.com) is the place where all the
+      community buzz happens. It's a place to get the answer to your technical
```
[the remark this answers is not in the document]
Comment (ArjaanBuijk) [reply domain.yml]: Yesss, good change. I included it.

File: domain.yml [hunk 7]
```diff
@@ -269,88 +286,134 @@ templates:
...
+  - text: >-
+      Rasa provides machine learning tools for developers to create contextual
+      AI assistants and chatbots! You can read more about it here in the docs
+      or on our [website](https://rasa.com/).
   utter_ask_whatspossible:
-  - text: You can book a sales call or subscribe to our newsletter.
+  - text: |
+      You can ask me about:
+      - how to get started with Rasa 
+      - the components of Rasa
```
[the remark this answers is not in the document]
Comment (ArjaanBuijk) [reply domain.yml]: I will include your suggestion. Makes more sense to me as well.

File: domain.yml [hunk 8]
```diff
@@ -400,69 +480,85 @@ templates:
...
   utter_get_started_step2:
-  - text: I can show you how to start building your first AI assistant with Rasa. Are you ready?
+  - text: You can try out the Rasa Stack directly here on this page or on 
+      our local machine. I'm here to help if you have any questions. 
+      \n I'm still learning though, so I may not have all the answers.
   utter_get_started_step3:
-  - text: Now, let's take a look at Rasa X - the tool that helps Rasa learn from real
-      conversations. Are you ready?
+  - text: Now it is time to start building the first bot yourself. I can help 
+      you with the installation.
```
[the remark this answers is not in the document]
Comment (ArjaanBuijk) [reply domain.yml]: Ah, I see!  I will review those and revert the change.

File: domain.yml [hunk 9]
```diff
@@ -542,31 +645,47 @@ templates:
...
   - text: Rasa is an open source machine learning framework for automated text and voice-based conversations.
       Rasa X is a tool that helps you build, improve, and deploy AI Assistants that are powered by the Rasa framework.
   utter_rasa_cost:
-  - text: Rasa is free!
+  - text: >-
+      The Rasa Stack is open source and completely free to use. You can get
+      started [here](https://rasa.com/docs/get_started_step1/). 
+      
+      Rasa Platform is our enterprise product, find out more
+      [here](https://rasa.com/products/rasa-platform/).
```
[the remark this answers is not in the document]
Comment (ArjaanBuijk) [reply domain.yml]: I think that someone who asks about costs is probably interested in finding out the different offerings. What about something like this, including a link to the pricing page, which has a link to `contact sales`

---
The Rasa machine learning framework is open source.

In addition to that, we offer a toolset [Rasa X](https://rasa.com/docs/rasa-x/), which comes in two editions: 
- Rasa X Community Edition (Free)
- Rasa X Enterprise Edition (Paid Subscription)

You can find out more [here](https://rasa.com/product/pricing/) about the differences between the Rasa X editions.

File: domain.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (erohmensing) [reply domain.yml]: Coolio, sounds good. Didn't know if she was immediately going live all over or just on the docs

File: domain.yml (hunk 9 above)
[the remark this answers is not in the document]
Comment (erohmensing) [reply domain.yml]: Sounds good to me!

**Comment by erohmensing** [comment]
Sorry, I should have merged arjaan's changes before adding more, this is getting big. But i think everything here is ready except for updating the config with embeddingpolicy

**Pull request merged**